library/src/main/java/me/toptas/fancyshowcase/FancyShowCaseView.java

```diff
@@ -453,6 +454,35 @@ protected void setDismissListener(DismissListener dismissListener) {
         mDismissListener = dismissListener;
     }
 
+    /**
+     * Returns if current shape has a radius. Have to add more cases if new shapes doesn't have radius
+     *
+     * @return true if current shape has a radius
+     */
+    public boolean hasRadius(){
```

Comment:
This method is unnecessary. `getFocusRadius()` will return `-1` if there is no focus.